.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
> I think that the untranspiled version should not be published to npm, at least not until the vast majority of javascript engines support ES6. That probably means waiting for a couple of years.

I disagree with this. The cast majority of JavaScript engines already supports ES6 (Chrome, Firefox, node 4+).

However, since module bloat seems to be a problem among the node community (with typically deep dependency trees), I'm for leaving `src` out.